<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Workfolder</class>
 <widget class="QWidget" name="Workfolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workfolder window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#workfolderFrame{
background: rgb(214, 219, 233,255);
}

QListView{
background: rgb(214, 219, 233,255);
border: 2px solid rgb(231, 236, 250, 255);
}

QPushButton:hover{
background: rgb(255, 255, 191,255);
border:  2px solid rgb(255, 170, 0,255);
}

QPushButton{
border:  2px solid rgb(231, 236, 250, 255);;
background: rgb(214, 219, 233,255);
}

QTableWidget{
background: rgb(214, 219, 233,255);
border: 2px solid rgb(231, 236, 250, 255);
}

QHeaderView::section{
background-color: rgb(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="workfolderFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="5,3">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="120">
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonOk">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSelectOther">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select other</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonSelectOther</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>released()</signal>
   <receiver>Workfolder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectOther</sender>
   <signal>released()</signal>
   <receiver>Workfolder</receiver>
   <slot>on_push_button_select_other()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectOther</sender>
   <signal>released()</signal>
   <receiver>Workfolder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_push_button_select_other()</slot>
 </slots>
</ui>
